src/models/differentiable/other/evotree.jl

```diff
@@ -35,7 +36,7 @@ end
 """
 Outer constructor method for EvoTreeModel.
 """
-function EvoTreeModel(model::Any; likelihood::Symbol=:classification_binary)
+function EvoTreeModel(model; likelihood::Symbol=:classification_binary)
     return EvoTreeModel(model, likelihood)
 end
```

Comment:
what is the purpose of this constructor? it's just confusing if in one constructor `likelihood` is a regular param and in another one it's a kwarg.

Reply:
The purpose is being able to define a model without specifying the likelihood. It might be clearer if likelihood is just the default argument here rather than a keyword argument, but right now, each model class has an outer constructor that works like this and has likelihood as a kwarg, so fixing it would require quite a lot of refactoring. Do you think this issue is serious enough to be worth spending time on?

Reply:
no